crates/testing/tests/tests_1/proposal_ordering.rs

```diff
@@ -100,14 +100,17 @@ async fn test_ordering_with_specific_order(input_permutation: Vec<usize>) {
         ),
     ];
 
-    let view_2_inputs = permute_input_with_index_order(inputs, input_permutation);
+    let mut view_2_inputs = permute_input_with_index_order(inputs, input_permutation);
+    view_2_inputs.insert(0, DACertificateRecv(dacs[1].clone()));
+    view_2_inputs.insert(0, VIDShareRecv(get_vid_share(&vids[2].0, handle.get_public_key())));
+    view_2_inputs.insert(0, VIDShareRecv(get_vid_share(&vids[1].0, handle.get_public_key())));
```

Comment:
Why do we need the VID shares for two views?

Reply:
We now need VID to propose so vids[2] is so we can call `Header::new()` with the vid common info to create a proposal.  vids[1] is needed to complete the prior rounds validation for the proposal and updating the state map.